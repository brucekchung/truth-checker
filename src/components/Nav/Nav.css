.Navi {
  display: flex;
  justify-content: space-around;
  background: #535252;
  padding-top: 12px;
  padding-bottom: 12px;
}

.Navi a {
/*  color: white;*/
/*  text-decoration: none;*/
}

.Navi a:hover {
  
}

.Nav {
  color: white;
  text-decoration: none;
}

.active {
  color: #88b4cc;
}

