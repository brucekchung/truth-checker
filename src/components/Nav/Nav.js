import React from 'react'
import { NavLink } from 'react-router-dom'
import './Nav.css'

export const Nav = () => (
  <div className='Navi'>
    <NavLink className="Nav" exact to='/'>About</NavLink>
    <NavLink className="Nav" to='/Methods'>Methods</NavLink>
    <NavLink className="Nav" to='/Result'>Result</NavLink>
  </div>
)