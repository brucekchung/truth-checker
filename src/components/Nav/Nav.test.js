import React from 'react'
import { Nav } from './Nav'
import { shallow } from 'enzyme'

describe('Nav', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Nav />)

    expect(wrapper).toMatchSnapshot()
  })
})