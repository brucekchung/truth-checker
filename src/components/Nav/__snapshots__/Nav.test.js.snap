// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
>
          About
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/Methods"
>
          Methods
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/Result"
>
          Result
</NavLink>,
      ],
      "className": "Nav",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "About",
          "to": "/",
        },
        "ref": null,
        "rendered": "About",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Methods",
          "to": "/Methods",
        },
        "ref": null,
        "rendered": "Methods",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Result",
          "to": "/Result",
        },
        "ref": null,
        "rendered": "Result",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/"
>
            About
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/Methods"
>
            Methods
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/Result"
>
            Result
</NavLink>,
        ],
        "className": "Nav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "About",
            "to": "/",
          },
          "ref": null,
          "rendered": "About",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Methods",
            "to": "/Methods",
          },
          "ref": null,
          "rendered": "Methods",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Result",
            "to": "/Result",
          },
          "ref": null,
          "rendered": "Result",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
